modules/33acrossBidAdapter.js

```diff
@@ -143,14 +143,30 @@ function _createServerRequest(bidRequest, gdprConsent) {
 }
 
 // Sync object will always be of type iframe for TTX
-function _createSync(siteId) {
+function _createSync({siteId, gdprConsent = {}}) {
   const ttxSettings = config.getConfig('ttxSettings');
   const syncUrl = (ttxSettings && ttxSettings.syncUrl) || SYNC_ENDPOINT;
 
-  return {
+  const {consentString, gdprApplies} = gdprConsent;
+
+  const sync = {
     type: 'iframe',
     url: `${syncUrl}&id=${siteId}`
+  };
+
+  const gdprMap = {
+    'true': 1,
+    'false': 0
+  };
+
+  if (typeof gdprApplies === 'boolean') {
+    const gdpr = gdprMap[gdprApplies.toString()];
+    sync.url = `${sync.url}&gdpr=${gdpr}`;
   }
+
+  sync.url = `${sync.url}&gdpr_consent=${consentString}`;
```

Comment:
try to use the util function `parseQueryStringParameters()` provided by prebidjs (https://github.com/33Across/Prebid.js/blob/master/src/utils.js), it does the encoding on the value, but do we need the encoding on the `consentString`??

Reply:
Not using the util since it does not work well with the expected URL structure. However I am directly encoding the consent string using `encodeURIComponent`